Library/Homebrew/cmd/doctor.rb

```diff
@@ -435,6 +443,7 @@ def check_access_usr_local
 
     You should probably change the ownership and permissions of /usr/local
     back to your user account.
+      sudo chown $(whoami):admin /usr/local
```

Comment:
`/usr/local` itself is still better to be owned by admin. So it would complain permission problem when we add new files(like `.travis.yml` we just added recently) in the root.